autolens/plot/plotters/fit_imaging_plotters.py

```diff
@@ -0,0 +1,292 @@
+        visuals_2d: lensing_visuals.Visuals2D = lensing_visuals.Visuals2D(),
+        include_2d: lensing_include.Include2D = lensing_include.Include2D(),
+    ):
+
+        super().__init__(
+            fit=fit,
+            mat_plot_2d=mat_plot_2d,
+            include_2d=include_2d,
+            visuals_2d=visuals_2d,
+        )
+
+    @property
+    def visuals_with_include_2d(self) -> "vis.Visuals2D":
+        """
+        Extracts from a `Structure` attributes that can be plotted and return them in a `Visuals` object.
+
+        Only attributes with `True` entries in the `Include` object are extracted for plotting.
+
+        From an `AbstractStructure` the following attributes can be extracted for plotting:
+
+        - origin: the (y,x) origin of the structure's coordinate system.
+        - mask: the mask of the structure.
+        - border: the border of the structure's mask.
+
+        Parameters
+        ----------
+        structure : abstract_structure.AbstractStructure
+            The structure whose attributes are extracted for plotting.
+
+        Returns
+        -------
+        vis.Visuals2D
+            The collection of attributes that can be plotted by a `Plotter2D` object.
+        """
+
+        visuals_2d = super(FitImagingPlotter, self).visuals_with_include_2d
+
+        visuals_2d.mask = None
+
+        return visuals_2d + lensing_visuals.Visuals2D(
+            light_profile_centres=self.extract_2d(
+                "light_profile_centres", self.tracer.planes[0].light_profile_centres
+            ),
+            mass_profile_centres=self.extract_2d(
+                "mass_profile_centres", self.tracer.planes[0].mass_profile_centres
+            ),
+            critical_curves=self.extract_2d(
+                "critical_curves", self.tracer.critical_curves
+            ),
+        )
+
+    @property
+    def tracer(self):
+        return self.fit.tracer
+
+    @property
+    def tracer_plotter(self):
+        return ray_tracing_plotters.TracerPlotter(
+            tracer=self.tracer,
+            grid=self.fit.grid,
+            mat_plot_2d=self.mat_plot_2d,
+            visuals_2d=self.visuals_2d,
+            include_2d=self.include_2d,
+        )
+
+    @property
+    def inversion_plotter(self):
+        return inversion_plotters.InversionPlotter(
+            inversion=self.fit.inversion,
+            mat_plot_2d=self.mat_plot_2d,
+            visuals_2d=self.tracer_plotter.visuals_with_include_2d_of_plane(
+                plane_index=1
+            ),
+            include_2d=self.include_2d,
+        )
+
+    @abstract_plotters.for_figure_with_index
+    def figure_subtracted_image_of_plane(self, plane_index):
+        """Plot the model image of a specific plane of a lens fit.
+    
+        Set *autolens.datas.arrays.plotter.plotter* for a description of all input parameters not described below.
+    
+        Parameters
+        -----------
+        fit : datas.fitting.fitting.AbstractFitter

```

Comment:
If you specify types in the signature there's no need to put types in the docs

Reply:
Also these docs don't match the function signature. Your PyCharm linter should indicate that there's an issue. 

Pro tip: if you make your linter green before every commit it will force you to fix these issues.